app/controllers/mentors/comments_controller.rb

```diff
@@ -20,7 +20,7 @@ def create_params
     end
 
     def update_params
-      params.require(:mentor_comment).permit(:text)
+      params.require(:mentor_comment).permit(:id, :text)
```

Comment:
Since this potentially allows to edit other user's comments (it shouldn't in this case, since we fetch the comments scoped by commenter in `mentor_comments`) I would like to have a test that catches a regression: can you add a test example where the a user attempts to update some other user's mentor comment please?